### DISCUSSION TITLE
Please tell me the relation of Corinna and the trial of the class implementation.

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [Ovid/Cor#66] [Q&A] [answered]
Paul seems to implement class feature.

https://github.com/leonerd/perl5/tree/feature-class/

It seems that the implementation is based on Corinna Specification.

https://github.com/Ovid/Cor

Could you tell me the relation of Corinna and the trial of the class implementation?

**Comment by haarg** [comment, marked as answer]
The class feature that LeoNerd is working on is intended to be the implementation of the Corinna spec.

**Reply by yuki-kimoto** [reply]
Thanks.

**Reply by Ovid** [reply]
This is correct, but also note that the feature class work is a subset of Corinna. See this P5P thread on the pre-RFC: https://www.nntp.perl.org/group/perl.perl5.porters/2022/01/msg262335.html

**Reply by yuki-kimoto** [reply]
Thanks.
